jetty-server/src/main/java/org/eclipse/jetty/server/handler/HandlerList.java

```diff
@@ -44,7 +44,7 @@ public void handle(String target, Request baseRequest, HttpServletRequest reques
     {
         Handler[] handlers = getHandlers();
 
-        if (handlers != null && isStarted())
```

Comment:
This potentially controversial, as in order for the example in the PR to work, this `isStarted()` check cannot be here.

This seems arbitrary and not all Handler do this check.
Eg: `ContextHandlerCollection` and `GzipHandler` do not check `isStarted()`, BUT `HandlerList`, `HandlerCollection`, and `RewriteHandler` do.

Reply:
I agree that checking is a bit arbitrary, but the check should be valid non-the-less